source/hardware/controllers/mvave_smc_mixer.rst

```diff
@@ -0,0 +1,116 @@
+M-Vave SMC-Mixer
+================
+
+`Manufacturer's product page <https://www.cuvave.com/productinfo/1106102.html>`_ ·
+`Forum thread <https://mixxx.discourse.group/t/m-wave-sinco-smc-mixer-radio-broadcast-mapping/30366>`_
+
+.. versionadded:: 2.5.1
+
+Overview
+--------
+
+.. figure:: ../../_static/controllers/mvave_smc_mixer.svg
+   :align: center
+   :width: 100%
+   :figwidth: 100%
+   :alt: M-Vave SMC-Mixer
+   :figclass: pretty-figures
+
+See the tables below for controls descriptions.
+
+.. note:: The mixer can be connected over Bluetooth, USB-C, or both. For use with Mixxx it is recommended to disable Bluetooth and connect only via USB-C.
+
+Deck and EQ
+-----------
+
+On the left side of the controller each set of a fader, rotary encoder, and
+buttons affect a single deck starting with 3 on the left, then 1, 2, and finally
+deck 4.
+
+..
+
+.. csv-table::
+   :header: "Control", "Name", "Function"
+   :widths: 5 25 70
+
+   "Knob", "(rotate)", "Adjust selected function (defaults to channel gain)"
+   "Treble", ":hwlabel:`M` (press)", "Toggle high filter cutoff"
+   "Treble", ":hwlabel:`M` (long press)", "Toggle knob between gain and high filter"
+   "Mid", ":hwlabel:`S` (press)", "Toggle mid filter cutoff"
+   "Mid", ":hwlabel:`S` (long press)", "Toggle knob between gain and mid filter"
+   "Bass", ":hwlabel:`R` (press)", "Toggle low filter cutoff"
+   "Bass", ":hwlabel:`R` (long press)", "Toggle knob between gain and low filter"
+   "Quick Effect", ":hwlabel:`□` (press)", "Toggle quick effect rack"
+   "Quick Effect", ":hwlabel:`□` (long press)", "Toggle knob between gain and quick effect"
+   "LEDs", "", "Blinks when the hardware control does not match the software control and soft takeover is enabled"
+   "Faders", "", "Adjust volume level"
+
+Beat Grid
+---------
+
+The right hand side of the controller is mapped to controls relating to the
+tempo and beat grid.
+Each set of controls is mapped to one deck and follows the same order (3, 1, 2,
+4) as the left side controls.
+The only exception is the rotary encoders, which are mapped separately as
+described below.
+
+.. csv-table::
+   :header: "Control", "Name", "Function"
+   :widths: 5 25 70
+
+   "Slip", ":hwlabel:`M`", "Toggle slip mode"
+   "Quantize", ":hwlabel:`S`", "Toggle quantize mode"
+   "Key Lock", ":hwlabel:`R`", "Toggle the key lock"
+   "Cue", ":hwlabel:`□`", ":term:`Headphone button` / PFL. Toggle sending channel to the cue (headphone) channel"
+   "LEDs", "", "Blinks when the hardware control does not match the software control and soft takeover is enabled"
+   "Faders", "", "Adjust track tempo"
+
+Transports
+----------
+
+The transport buttons along the bottom affect only the currently selected deck,
+with the exception of "record" which toggles recording of the mix and the
+navigation buttons which always affect the library view.
```

Comment:
Maybe mention that the limitation applies that the window needs to be focused?

Reply:
Done, thanks!